Mathlib/AlgebraicTopology/SimplicialSet/StrictSegal.lean

```diff
@@ -86,6 +87,152 @@ theorem spineToSimplex_edge (f : Path X n) (j l : ℕ) (hjl : j + l ≤ n) :
   unfold diagonal
   simp only [← FunctorToTypes.map_comp_apply, ← op_comp, diag_subinterval_eq]
 
+/-- For any `σ : X ⟶ Y` between `StrictSegal` simplicial sets, `spineToSimplex`
+commutes with `Path.map`. -/
+lemma spineToSimplex_map {X Y : SSet.{u}} [StrictSegal X] [StrictSegal Y]
+    {n : ℕ} (f : Path X (n + 1)) (σ : X ⟶ Y) :
+    spineToSimplex (f.map σ) = σ.app _ (spineToSimplex f) := by
+  apply spineInjective
+  ext k
+  dsimp only [spineEquiv, Equiv.coe_fn_mk, Path.map, spine_arrow]
+  rw [← types_comp_apply (σ.app _) (Y.map _), ← σ.naturality]
+  simp only [types_comp_apply, spineToSimplex_arrow]
+
+/-- If we take the path along the spine of a face of a `spineToSimplex`, the
+common vertices will agree with those of the original path. In particular, the
+vertices `j < i` can be identified with the same vertices in the original path. -/
+lemma spine_δ_vertex_lt (f : Path X (n + 1)) {i : Fin (n + 2)} {j : Fin (n + 1)}
```

Comment:
What is the naming pattern for the `lt/gt/le/ge/eq` suffices? Okay, `eq` is clear.

In
```
lemma mkOfSucc_δ_lt {n : ℕ} {i : Fin n} {j : Fin (n + 2)}
    (h : i.succ.castSucc < j) :
    mkOfSucc i ≫ δ j = mkOfSucc i.castSucc :=
```
The LHS of the conclusion has `i` appearing before `j`, and the condition is `i + 1 < j`. So that could be a reason to call this version `_lt`.
But in the lemma at hand, `i` appears before `j` in the conclusion, and the condition is `i > j`. So I would expect this version to be called `gt`.

Are you using a different convention?

Reply:
I don't have a strong opinion either.

Reply:
I think I slightly prefer the geometric intuition of `_lt` implying `arrow index < face index`. I have updated some comments to more directly indicate this convention. I also reordered and renamed the arguments to `spine_δ_*` to better match those of  `mkOfSucc_δ_*`.

I will leave this comment unresolved in case anyone else has the opposite opinion on the convention here.